setup/setup.py

```diff
@@ -69,11 +72,10 @@
 GIT_BRANCH_NAME = check_output(shlex.split(
   'git rev-parse --abbrev-ref HEAD')).strip()
 GIT_HEAD_REV = check_output(shlex.split('git rev-parse --short HEAD')).strip()
-VERSION = check_output(shlex.split('cat {}/version.txt'.format(CUT_PATH))).strip()
```

Comment:
You could

```
VERSION = check_output(shlex.split('cat {}/version.txt'.format(CUT_PATH))).strip()
try:
    VERSION = VERSION.decode('utf-8')
except:
    pass
```

To avoid the so complex logic to keep compatible with python 2/3.

Reply:
Thanks for your reply.  
Keeping compatible with python2 and 3 is not complicated. I just replacee the version.txt file with `_VERSION` member in setup.py. And it makes decoding `version` more simple. The complex logic in Makfile is to support `gl.__version__`.